lesson08/assignment/video_cli/l8/src/form.vue

```diff
@@ -43,7 +43,8 @@
     methods: {
       submitForm() {
         // form submission logic here
-        if (/* form submission is successful */) {
+        /* form submission is successful */
+        if (this.formSubmitted) {
```

Comment:
This isn't technically what you want here for your form, but having the comment inside the if() results in a syntax error, preventing the pages from rendering.